environment.yml

```diff
@@ -4,12 +4,12 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python
+  - python==3.11
   - pip
   - python-dotenv
   - pytest
   - lxml
-  - pandas
+  - pandas<2.2
```

Comment:
Could you provide a comment of why we can't update beyond pandas 2.2?

Reply:
The dependency versions were pulled from Xander's master branch. Xander would be the best person to add a comment for this.